pkgs/development/libraries/wxGTK-2.8/default.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv, fetchurl, pkgconfig, gtk2, libXinerama, libSM, libXxf86vm, xf86vidmodeproto
 , gstreamer, gst_plugins_base, GConf, libX11, cairo
-, withMesa ? true, mesa ? null, compat24 ? false, compat26 ? true, unicode ? true,
+, withMesa ? true, mesa ? null, compat24 ? false, compat26 ? true, unicode ? true
+, Carbon ? null, Cocoa ? null, CoreServices
```

Comment:
You didn't add a default value for CoreServices, but this isn't really necessary. If those are not evaluated in the derivation it's fine to pass those around on other platforms.